behavior_metrics/utils/environment.py

```diff
@@ -114,17 +114,17 @@ def close_ros_and_simulators():
         except subprocess.CalledProcessError as ce:
             logger.error("SimulatorEnv: exception raised executing killall command for CarlaUE4-Linux-Shipping {}".format(ce))
 
-    if ps_output.count('rosout') > 0:
-        try:
-            import rosnode
-            for node in rosnode.get_node_names():
-                if node != '/carla_ros_bridge':
-                    subprocess.check_call(["rosnode", "kill", node])
-
-            logger.debug("SimulatorEnv:rosout killed.")
-        except subprocess.CalledProcessError as ce:
-            logger.error("SimulatorEnv: exception raised executing killall command for rosout {}".format(ce))
-    
+    # if ps_output.count('rosout') > 0:
```

Comment:
These lines are intended to kill all the possible pending processes for ROS, CARLA, Gazebo, CARLA-ROS-brigde. They are at least needed for the CARLA part. What's happening with this behavior in Gazebo?